src/main/java/edu/greenblitz/robotName/utils/shootingCalculations/ShooterAngle.java

```diff
@@ -0,0 +1,23 @@
+package edu.greenblitz.robotName.utils.shootingCalculations;
+
+import edu.greenblitz.robotName.FieldConstants;
+import edu.greenblitz.robotName.utils.GBMath;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.geometry.Translation3d;
+
+public class ShooterAngle {
+
+    public static Rotation2d getShooterAngleBasedOnPosition(Translation3d position) {
```

Comment:
change to translation 3d to a trnaslation 2d. we dont need to z axis for the calculaitons since its a constant